test/unit/BPool/BPool_Bind_Unbind.tree

```diff
@@ -0,0 +1,51 @@
+BPool::bind
+├── when reentrancy lock is NOT set
+│   ├── when called NOT by controller
```

Comment:
`when caller is NOT controller` and `when caller is controller`?
`called NOT` is weird phrasing